### ISSUE TITLE
[NG] Migrate the site data for the Resources page

### ISSUE THREAD

**Issue by chandrashritii** [layer5io/layer5#713]
**Description**
Currently, the [Resources page](https://layer5ng.netlify.app/learn) on the [layer5-ng site](https://layer5ng.netlify.app/) points to a 404 error. 

**Expected Behavior**

The data from the [Resources page](https://layer5.io/learn) on the [layer5.io site](https://layer5.io/) needs to be migrated to the layer5-ng site. 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

**Comment by iambami** [comment]
I will love to do this

**Comment by PluckyPrecious** [comment]
Oh great 👍🏽, do go ahead.

**Comment by vinayaksh42** [comment]
Can i work on this issue as the last assigned person has removed their assignment?
@PluckyPrecious @leecalcote

**Comment by leecalcote** [comment]
@vinayaksh42 that would be great, yes. 👍

**Comment by Tanuj22** [comment]
@vinayaksh42 sorry I didn't have a look at this before. The books part of the page is already migrated and the workshop page as already someone assigned to it.

**Comment by Tanuj22** [comment]
Blocked by #781

**Comment by vinayaksh42** [comment]
> @vinayaksh42 sorry I didn't have a look at this before. The books part of the page is already migrated and the workshop page as already someone assigned to it. 

No problem , so basically this issue is not required anymore?

**Comment by Tanuj22** [comment]
It can be worked on after the workshop part is done 

> > @vinayaksh42 sorry I didn't have a look at this before. The books part of the page is already migrated and the workshop page as already someone assigned to it.
> 
> No problem , so basically this issue is not required anymore?

**Comment by vinayaksh42** [comment]
> It can be worked on after the workshop part is done 
> 
> > > @vinayaksh42 sorry I didn't have a look at this before. The books part of the page is already migrated and the workshop page as already someone assigned to it.
> > 
> > No problem , so basically this issue is not required anymore?
> 
> 

Okay

**Comment by Tanuj22** [comment]
If you are comfortable with gatsby would you like to work on making the careers page dynamic?

**Comment by vinayaksh42** [comment]
> If you are comfortable with gatsby would you like to work on making the careers page dynamic? 

Not exactly comfortable but I think we can give it a go

**Comment by Tanuj22** [comment]
@kishore007k do you want a send a quick PR for this? As the books and Workshop page is already done

**Comment by kishore007k** [comment]
Just to be clear. This page should contain both the workshops and book page contents?

**Comment by Tanuj22** [comment]
Yes

**Comment by kishore007k** [comment]
Ok, I could do that.

**Comment by Tanuj22** [comment]
@kishore007k one thing to note. In your PR you have used a page query to fetch the workshops in the pages/workshops.js file. What you can do is that shift this query to the `Workshop-gird` section you had created as a static query. (check the programs section for reference) and use this section directly in the `/learn` page. Instead of rewriting the query. You the same thing for Books

**Comment by kishore007k** [comment]
Could you assign this to me! @Tanuj22
